examples/src/pages/Toolbar.tsx

```diff
@@ -0,0 +1,242 @@
+/*
```

Comment:
I think we should just extend the Migration-example to also have a toolbar :)

Reply:
Merged into Migration-example